### PR TITLE
Add parameters and doc for RandomForestClassifier ctor

### PR DESCRIPTION
**Pull request by wbo4958**

### CODE REVIEW

**Comment by wbo4958** [comment]
build

File: python/src/spark_rapids_ml/classification.py [hunk 1]
```diff
@@ -107,9 +107,86 @@ class RandomForestClassifier(
...
+
+    Parameters
+    ----------
+
+    featuresCol:
+        The feature column names, spark-rapids-ml supports vector, array and columnar as the input.\n
+            * When the value is string, the feature columns must be assembled into 1 column with vector or array type.
+            * When the value is a list of string, the feature columns must be numeric types.
+    labelCol:
+        The lable column name.
```
Comment (leewyang) [inline python/src/spark_rapids_ml/classification.py:123]: typo: "lable" -> "label"

File: python/src/spark_rapids_ml/classification.py [hunk 2]
```diff
@@ -107,9 +107,86 @@ class RandomForestClassifier(
...
+    min_samples_split:
+        The minimum number of samples required to split an internal node.\n
+         * If type ``int``, then ``min_samples_split`` represents the minimum
+           number.
+         * If type ``float``, then ``min_samples_split`` represents a fraction
+           and ``ceil(min_samples_split * n_rows)`` is the minimum number of
+           samples for each split.    max_samples:
+        Ratio of dataset rows used while fitting each tree.
+    max_leaves:
+        Maximum leaf nodes per tree. Soft constraint. Unlimited, If -1.
```
Comment (leewyang) [inline python/src/spark_rapids_ml/classification.py:185]: "Unlimited, if -1" (capitalization)

**Review by leewyang — COMMENTED** [review]
Minor typos

File: python/src/spark_rapids_ml/classification.py (hunk 2 above)
Comment (wbo4958) [reply python/src/spark_rapids_ml/classification.py]: Done. Thx

File: python/src/spark_rapids_ml/classification.py (hunk 1 above)
Comment (wbo4958) [reply python/src/spark_rapids_ml/classification.py]: Done. Thx

**Review by leewyang — APPROVED** [review]

**Comment by wbo4958** [comment]
build

**Pull request merged**